vocabularies/Common.md

```diff
@@ -29,86 +29,87 @@ Term|Type|Description
 [SecondaryKey](./Common.xml#L206:~:text=<Term%20Name="-,SecondaryKey,-") *(Deprecated)*|\[PropertyPath\]|<a name="SecondaryKey"></a>Use term `AlternateKeys` from the OASIS Core vocabulary instead
 [MinOccurs](./Common.xml#L218:~:text=<Term%20Name="-,MinOccurs,-")|Int64|<a name="MinOccurs"></a>The annotated set or collection contains at least this number of items
 [MaxOccurs](./Common.xml#L222:~:text=<Term%20Name="-,MaxOccurs,-")|Int64|<a name="MaxOccurs"></a>The annotated set or collection contains at most this number of items
-[AssociationEntity](./Common.xml#L226:~:text=<Term%20Name="-,AssociationEntity,-")|\[NavigationPropertyPath\]|<a name="AssociationEntity"></a>Entity representing an n:m association with attributes
```

Comment:
This Common.md file is derived from one of the others? Which one is the original? (Because this diff is too large, I would like to just review your manual changes)

Reply:
Common.xml is the original.